redis/config_test.go

```diff
@@ -0,0 +1,102 @@
+package redis
+
+import (
+	"github.com/creasty/defaults"
+	"github.com/icinga/icinga-go-library/config"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestConfig(t *testing.T) {
+	var defaultOptions Options
+	if err := defaults.Set(&defaultOptions); err != nil {
+		require.NoError(t, err, "setting default options")
```

Comment:
Even when a potential error of `defaults.Set(&defaultOptions)` is not directly part of the tests, it will influence the test output. Most test cases using `Options: defaultOptions` would fail the `require.Equal(…)` check, resulting in somebody debugging the actual test, unaware that a precondition failed. Thus, i would like to fail early with a distinguishable error, easing the debugging for future eyes.

Reply:
Sure, the code is legit. But you could `s/err != nil/true/` w/o actually changing anything. Hence, the _conditional_ execution is a no-op.

Reply:
Now I am getting your point. Changed.